editor.planx.uk/src/themeOverrides.d.ts

```diff
@@ -25,10 +25,14 @@ declare module "@mui/material/styles" {
 declare module "@mui/material/styles/createPalette" {
   interface Palette {
     border: { main: string; input: string; light: string };
+    link: { main: string; };
+    prompt: { main: string; contrastText: string; light: string; dark: string; };
```

Comment:
I'll be honest, not a huge fan of `prompt` but it was the best worst option here 😅 Very open to other suggestions.

"Action" is already used by MUI in a different context, "Button" feels a bit odd / meaningless - 

```jsx
<Button color="button">
  Continue
</Button>
```